module-3/pop/Element.js

```diff
@@ -14,4 +14,41 @@
  *       in it's children (recursively) the Element with
  *       the given name or throws an Erorr if it cannot
  *       find the element
- */
\ No newline at end of file
+ */
+
+const ElementFinder = require("../test/mock/ElementFinder");
```

Comment:
It is enough to import the `element` static method, and it will create the instance of the `ElementFinder`

Reply:
After I changed this line to this:
_import { element } from "../test/mock/ElementFinder"_
I got an error:  _Cannot use import statement outside a module_
Is there any other way to import just the _element_ static method?

Reply:
you have to initialize into a variable 
const { element } = require("../test/mock/ElementFinder");